src/components/StyleGuide/Colors.stories.mdx

```diff
@@ -6,82 +6,377 @@ import SimpleGrid from "../Grid/SimpleGrid";
+        dataWhite: ["1.88", "Fail", "Fail"],
+        textColor: "ui.black",
+      },
+      lightCool: {
+        dataBlack: ["17.30", "AAA", "AAA"],
+        dataWhite: ["1.21", "Fail", "Fail"],
+        textColor: "ui.black",
+      },
+      xLightCool: {
+        dataBlack: ["19.26", "AAA", "AAA"],
+        dataWhite: ["1.09", "Fail", "Fail"],
+        textColor: "ui.black",
+      },
+      xxLightCool: {
+        dataBlack: ["20.12", "AAA", "AAA"],
+        dataWhite: ["1.04", "Fail", "Fail"],
+        textColor: "ui.black",
+      },
+      lightWarm: {
+        dataBlack: ["17.98", "AAA", "AAA"],
+        dataWhite: ["1.17", "Fail", "Fail"],
+        textColor: "ui.black",
+      },
+      xLightWarm: {
+        dataBlack: ["19.76", "AAA", "AAA"],
+        dataWhite: ["1.06", "Fail", "Fail"],
+        textColor: "ui.black",
+      },
+    },
+    white: {
+      dataBlack: ["21.00", "AAA", "AAA"],
+      dataWhite: ["1.00", "Fail", "Fail"],
+      textColor: "ui.black",
+    },
+  },
+};
 export const cssVars = {
-  brand: ["--nypl-colors-brand-primary", "--nypl-colors-brand-secondary"],
+  brand: [
+    {
+      name: "brand.primary",
+      dataBlack: defaultData.brand.primary.dataBlack,
+      dataWhite: defaultData.brand.primary.dataWhite,
+      textColor: defaultData.brand.primary.textColor,
+    },
+    {
+      name: "brand.secondary",
+      dataBlack: defaultData.brand.secondary.dataBlack,
+      dataWhite: defaultData.brand.secondary.dataWhite,
+      textColor: defaultData.brand.secondary.textColor,
+    },
+  ],
   blogs: [
-    "--nypl-colors-section-blogs-primary",
-    "--nypl-colors-section-blogs-secondary",
+    {
+      name: "section.blogs.primary",
+      dataBlack: defaultData.ui.gray.lightCool.dataBlack,
+      dataWhite: defaultData.ui.gray.lightCool.dataWhite,
+      textColor: defaultData.ui.gray.lightCool.textColor,
+    },
+    {
+      name: "section.blogs.secondary",
+      dataBlack: defaultData.ui.gray.medium.dataBlack,
+      dataWhite: defaultData.ui.gray.medium.dataWhite,
+      textColor: defaultData.ui.gray.medium.textColor,
+    },
   ],
   booksAndMore: [
-    "--nypl-colors-section-books-and-more-primary",
-    "--nypl-colors-section-books-and-more-secondary",
+    {
+      name: "section.books-and-more.primary",
+      dataBlack: defaultData.brand.primary.dataBlack,
+      dataWhite: defaultData.brand.primary.dataWhite,
+      textColor: defaultData.brand.primary.textColor,
+    },
+    {
+      name: "section.books-and-more.secondary",
+      dataBlack: defaultData.brand.secondary.dataBlack,
+      dataWhite: defaultData.brand.secondary.dataWhite,
+      textColor: defaultData.brand.secondary.textColor,
+    },
   ],
   education: [
-    "--nypl-colors-section-education-primary",
-    "--nypl-colors-section-education-secondary",
+    {
+      name: "section.education.primary",
+      dataBlack: ["3.94", "Fail", "AA"],
+      dataWhite: ["5.33", "AA", "AAA"],
+      textColor: "ui.white",
```

Comment:
Why is some information here and not in the `defaultData` array?

Reply:
I created `defaultData` to house objects that would be used more than once.  Right now, those that are not included in `defaultData` only appear that one time.  This follows the pattern that we’ve to handle typography styles.